static/js/level-detail.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * LevelDetail
+ *
+ * A bubble showing the level of a thing
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+'use strict';
+function LevelDetail(thing) {
+  this.thing = thing;
+}
+
+LevelDetail.prototype.attach = function() {
```

Comment:
What do you think about creating a ThingProperty base class and having LevelProperty extend ThingProperty? That's the approach I've been taking with the SmartPlug UI.